.travis.yml

```diff
@@ -3,10 +3,9 @@ language: php
 sudo: false
 
 php:
-  - 5.6
-  - 7.0
-  - 7.1
-  - 7.2
+  - 7.3
+  - 7.4
+  - 8.0
   - hhvm
```

Comment:
hhvm is not compatible to PHP any more.
We should remove hhvm.
https://hhvm.com/blog/2019/02/11/hhvm-4.0.0.html